deps/mbedtls.mk

```diff
@@ -49,7 +49,7 @@ $(MBEDTLS_OBJ_SOURCE): $(BUILDDIR)/mbedtls-$(MBEDTLS_VER)/Makefile
 
 $(BUILDDIR)/mbedtls-$(MBEDTLS_VER)/checked: $(MBEDTLS_OBJ_SOURCE)
 ifeq ($(OS),$(BUILD_OS))
-	$(MAKE) -C $(dir $@) test
+	-$(MAKE) -C $(dir $@) test
```

Comment:
I don't think this rule gets run by default unless you specifically request it, so please don't prefix it with `-`